src/index.js

```diff
@@ -15,26 +15,26 @@ const diplayLikes = () => {
       const titleLikes = document.getElementById(`title-like-${showId}`);
 
       if (titleLikes !== null) {
-        titleLikes.previousElementSibling.classList.add("text-danger");
+        titleLikes.previousElementSibling.classList.add('text-danger');
         titleLikes.innerHTML = result.likes;
       }
     });
   });
 };
 
 const recordLikeInteraction = () => {
-  const likes = document.querySelectorAll(".like");
+  const likes = document.querySelectorAll('.like');
   likes.forEach((like) => {
-    like.addEventListener("click", (e) => {
+    like.addEventListener('click', (e) => {
       recordLike(
         JSON.stringify({
           item_id: e.target.dataset.id,
-        })
+        }),
       );
-      e.target.classList.add("text-danger");
+      e.target.classList.add('text-danger');
 
       const titleLikes = document.getElementById(
-        `title-like-${e.target.dataset.id}`
+        `title-like-${e.target.dataset.id}`,
       );
       let numberOfLikes = parseInt(titleLikes.innerHTML, 10);
       // eslint-disable-next-line no-plusplus
```

Comment:
Let's remove all disabled Linters, Linting makes your code look cleaner and professional 🙂